Jenkinsfile

```diff
@@ -140,6 +140,13 @@ def daos_packages_version(String distro) {
     error "Don't know how to determine package version for " + distro
 }
 
+def fault_test_tag() {
+    if (skip_stage('fault-test')) {
+        return ",-faults"
+    }
+    return ""
+}
+
```

Comment:
Now that you have the `fault_status` tool, I wonder if this functionality should be done this way or if `Skip-fault-test: true` should just remove `--with-fault-injection` from the `scons` command and let the code in `launch.py` do the tagging fixup as it's been modified to do.

Reply:
The Skip-fault-test: pragma would only be useful now if the build enables faults and you wanted to run without tests that have the test_tag "faults"    Is this useful? ( maybe for debug) I am not sure CI would have a use for it and was thinking of removing the changes in Jenkinsfile

Reply:
Oh, no, by all means, please leave the pragma in, exactly for as you state, debugging issues with trying to build and test with FI disabled.  We'd not want to have to force an RC just to be able to do that.

But it does seem to me that the pragma should do everything required to disable faults (i.e. not even compiling them in), not just run the tests that needs it.

Reply:
Just a reminder that we have to resolve this also.

Reply:
Please review the latest changes.    It is not passing the initial ci checks and I am not sure i have the syntax correct.